ccl/partitionccl: TestValidIndexPartitionSetShowZones failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/cbe3d10fd28f7a1a1025028b9f840e2f6e437133

Parameters:
```
TAGS=
GOFLAGS=-race
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=809352&tab=buildLog

```
=== RUN   TestValidIndexPartitionSetShowZones
W180801 10:25:08.630680 108600 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180801 10:25:08.960926 108600 server/server.go:807  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180801 10:25:08.981984 108600 server/config.go:554  [n?] 1 storage engine initialized
I180801 10:25:08.982085 108600 server/config.go:557  [n?] RocksDB cache size: 128 MiB
I180801 10:25:08.982146 108600 server/config.go:557  [n?] store 0: in-memory, size 0 B
I180801 10:25:09.343869 108600 server/node.go:372  [n?] **** cluster 5dcd6fde-70bc-4c19-b0f2-f4a8035f4ac2 has been created
I180801 10:25:09.344031 108600 server/server.go:1382  [n?] **** add additional nodes by specifying --join=127.0.0.1:33103
I180801 10:25:09.345230 108600 gossip/gossip.go:363  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:33103" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > 
I180801 10:25:09.465767 108600 storage/store.go:1496  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180801 10:25:09.467244 108600 server/node.go:461  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7043.00 p25=7043.00 p50=7043.00 p75=7043.00 p90=7043.00 pMax=7043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180801 10:25:09.468736 108600 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180801 10:25:09.471612 108600 server/node.go:684  [n1] connecting to gossip network to verify cluster ID...
I180801 10:25:09.478008 108600 server/node.go:709  [n1] node connected via gossip and verified as part of cluster "5dcd6fde-70bc-4c19-b0f2-f4a8035f4ac2"
I180801 10:25:09.478493 108600 server/node.go:533  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180801 10:25:09.479687 108600 server/config.go:338  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180801 10:25:09.479770 108600 server/server.go:1814  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180801 10:25:09.482062 108600 server/server.go:1519  [n1] starting https server at 127.0.0.1:44389
I180801 10:25:09.482188 108600 server/server.go:1520  [n1] starting grpc/postgres server at 127.0.0.1:33103
I180801 10:25:09.482243 108600 server/server.go:1521  [n1] advertising CockroachDB node at 127.0.0.1:33103
W180801 10:25:09.502565 108600 sql/jobs/registry.go:299  [n1] unable to get node liveness: node not in the liveness table
I180801 10:25:09.625352 108836 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180801 10:25:10.008971 108883 storage/replica_command.go:279  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
W180801 10:25:10.100240 108496 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=0f56c79b key=/Table/SystemConfigSpan/Start rw=true pri=0.01369058 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119109.538874510,0 orig=1533119109.538874510,0 max=1533119109.538874510,0 wto=false rop=false seq=6
I180801 10:25:10.834965 108833 storage/replica_command.go:279  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180801 10:25:10.897327 108869 rpc/nodedialer/nodedialer.go:92  [replica consistency checker,n1,s1,r1/1:/{Min-System/}] connection to n1 established
I180801 10:25:10.908333 108494 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180801 10:25:11.287319 108903 storage/replica_command.go:279  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180801 10:25:11.730231 108890 storage/replica_command.go:279  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
W180801 10:25:11.769623 108863 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=9bd57f65 key=/Table/SystemConfigSpan/Start rw=true pri=0.00714129 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119111.218947685,0 orig=1533119111.218947685,0 max=1533119111.218947685,0 wto=false rop=false seq=12
I180801 10:25:12.096037 108652 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180801 10:25:12.222863 108948 storage/replica_command.go:279  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180801 10:25:12.572938 108934 storage/replica_command.go:279  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
W180801 10:25:12.717509 108845 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=7444d206 key=/Table/SystemConfigSpan/Start rw=true pri=0.05293163 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119112.146265321,0 orig=1533119112.146265321,0 max=1533119112.146265321,0 wto=false rop=false seq=6
I180801 10:25:12.819861 108843 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180801 10:25:13.331323 108967 storage/replica_command.go:279  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180801 10:25:13.501081 108929 storage/replica_command.go:279  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180801 10:25:13.743359 108956 storage/replica_command.go:279  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180801 10:25:13.931256 109014 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
W180801 10:25:13.985835 109029 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=a51c2f3e key=/Local/Range/Table/13/RangeDescriptor rw=true pri=0.00472860 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119113.743503794,0 orig=1533119113.743503794,0 max=1533119113.743503794,0 wto=false rop=false seq=1
W180801 10:25:14.001619 108909 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=a51c2f3e key=/Local/Range/Table/13/RangeDescriptor rw=true pri=0.00472860 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119113.743503794,0 orig=1533119113.743503794,0 max=1533119113.743503794,0 wto=false rop=false seq=1
W180801 10:25:14.091672 108986 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=a51c2f3e key=/Local/Range/Table/13/RangeDescriptor rw=true pri=0.00472860 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119113.743503794,0 orig=1533119113.743503794,0 max=1533119113.743503794,0 wto=false rop=false seq=1
I180801 10:25:14.177593 108988 storage/replica_command.go:279  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180801 10:25:14.295633 108998 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
W180801 10:25:14.377010 108994 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=526220f7 key=/Local/Range/Table/14/RangeDescriptor rw=true pri=0.00753449 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119114.177761563,0 orig=1533119114.177761563,0 max=1533119114.177761563,0 wto=false rop=false seq=1
W180801 10:25:14.386624 109059 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=526220f7 key=/Local/Range/Table/14/RangeDescriptor rw=true pri=0.00753449 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119114.177761563,0 orig=1533119114.177761563,0 max=1533119114.177761563,0 wto=false rop=false seq=1
I180801 10:25:14.465359 109046 storage/replica_command.go:279  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180801 10:25:14.572371 109032 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180801 10:25:14.713039 109077 storage/replica_command.go:279  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180801 10:25:14.826432 108600 server/server.go:1603  [n1] done ensuring all necessary migrations have run
I180801 10:25:14.826624 108600 server/server.go:1606  [n1] serving sql connections
I180801 10:25:14.941895 109083 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180801 10:25:15.026252 109085 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:33103} Attrs: Locality: ServerVersion:2.0-11} ClusterID:5dcd6fde-70bc-4c19-b0f2-f4a8035f4ac2 StartedAt:1533119109478125261 LastUp:1533119109478125261}
I180801 10:25:15.072602 109086 storage/replica_command.go:279  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180801 10:25:15.368846 109009 storage/replica_command.go:279  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180801 10:25:15.635068 109109 storage/replica_command.go:279  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180801 10:25:15.831751 109101 storage/replica_command.go:279  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180801 10:25:16.081138 109130 storage/replica_command.go:279  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180801 10:25:16.325994 109167 storage/replica_command.go:279  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180801 10:25:16.611644 109151 storage/replica_command.go:279  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180801 10:25:16.870952 109205 storage/replica_command.go:279  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180801 10:25:17.869648 109207 storage/replica_command.go:279  [split,n1,s1,r22/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r23]
I180801 10:25:17.907084 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "create_database", target: 52, info: {DatabaseName:d Statement:CREATE DATABASE d User:root}
I180801 10:25:18.002729 108734 storage/replica_proposal.go:222  [n1,s1,r7/1:/Table/{SystemCon…-11}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119117.982357854,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:18.177708 108730 storage/replica_proposal.go:222  [n1,s1,r10/1:/Table/1{3-4}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.123638453,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:18.212466 108775 storage/replica_proposal.go:222  [n1,s1,r22/1:/{Table/51-Max}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
W180801 10:25:18.217572 109286 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=e7a45c5f key=/Local/Range/Table/51/RangeDescriptor rw=true pri=0.00904039 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533119117.869811245,0 orig=1533119117.869811245,0 max=1533119117.869811245,0 wto=false rop=false seq=1
I180801 10:25:18.229537 108729 storage/replica_proposal.go:222  [n1,s1,r9/1:/Table/1{2-3}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.226016895,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:18.266404 108354 storage/replica_proposal.go:222  [n1,s1,r22/1:/{Table/51-Max}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:18.317969 108752 storage/replica_proposal.go:222  [n1,s1,r4/1:/System/{NodeLive…-tsd}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.314112910,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:18.543545 108786 storage/replica_proposal.go:222  [n1,s1,r5/1:/System/ts{d-e}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.539914970,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:18.629705 109119 storage/replica_command.go:279  [split,n1,s1,r23/1:/{Table/52-Max}] initiating a split of this range at key /Table/53 [r24]
I180801 10:25:18.747982 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "create_table", target: 53, info: {TableName:d.public.t Statement:CREATE TABLE t (c STRING PRIMARY KEY) PARTITION BY LIST (c) (PARTITION p0 VALUES IN ('a'), PARTITION p1 VALUES IN (DEFAULT)) User:root}
I180801 10:25:18.748260 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "create_table", target: 54, info: {TableName:d.public.t Statement:CREATE TABLE t (c STRING PRIMARY KEY) PARTITION BY LIST (c) (PARTITION p0 VALUES IN ('a'), PARTITION p1 VALUES IN (DEFAULT)) User:root}
I180801 10:25:19.086228 108766 storage/replica_proposal.go:222  [n1,s1,r23/1:/{Table/52-Max}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:19.091720 109174 storage/replica_command.go:279  [split,n1,s1,r24/1:/{Table/53-Max}] initiating a split of this range at key /Table/54 [r25]
I180801 10:25:19.198391 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 15, info: {Target:system.jobs Config: User:root}
I180801 10:25:19.296591 108788 storage/replica_proposal.go:222  [n1,s1,r24/1:/{Table/53-Max}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:19.503907 108805 server/status/runtime.go:424  [n1] runtime stats: 981 MiB RSS, 189 goroutines, 15 MiB/63 MiB/99 MiB GO alloc/idle/total, 36 MiB/82 MiB CGO alloc/total, 0.00cgo/sec, 0.00/0.00 %(u/s)time, 0.00 %gc (821x)
I180801 10:25:19.545256 108751 storage/replica_proposal.go:222  [n1,s1,r21/1:/Table/5{0-1}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119119.541393021,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:19.570995 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 16, info: {Target:.meta Config: User:root}
I180801 10:25:19.931769 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 22, info: {Target:.liveness Config: User:root}
I180801 10:25:20.694328 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 52, info: {Target:d Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:21.504814 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:22.329127 109346 storage/replica_command.go:279  [split,n1,s1,r25/1:/{Table/54-Max}] initiating a split of this range at key /Table/54/1 [r26]
I180801 10:25:22.352960 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t@primary Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:22.536249 108742 storage/replica_proposal.go:222  [n1,s1,r25/1:/{Table/54-Max}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:22.548350 109384 storage/replica_command.go:279  [split,n1,s1,r26/1:/{Table/54/1-Max}] initiating a split of this range at key /Table/54/2 [r27]
I180801 10:25:22.553448 108354 storage/replica_proposal.go:222  [n1,s1,r13/1:/Table/1{6-7}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119122.549789254,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:22.927889 108776 storage/replica_proposal.go:222  [n1,s1,r26/1:/{Table/54/1-Max}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:23.271122 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t.p0 Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:23.277055 109399 storage/replica_command.go:279  [split,n1,s1,r26/1:/Table/54/{1-2}] initiating a split of this range at key /Table/54/1/"a" [r28]
I180801 10:25:23.570711 108735 storage/replica_proposal.go:222  [n1,s1,r14/1:/Table/1{7-8}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119123.551636480,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:23.691081 108728 storage/replica_proposal.go:222  [n1,s1,r26/1:/Table/54/{1-2}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:23.731253 109186 storage/replica_command.go:279  [split,n1,s1,r28/1:/Table/54/{1/"a"-2}] initiating a split of this range at key /Table/54/1/"a"/PrefixEnd [r29]
I180801 10:25:24.216131 108785 storage/replica_proposal.go:222  [n1,s1,r28/1:/Table/54/{1/"a"-2}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0 following repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119118.200644259,0
I180801 10:25:24.483639 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 0, info: {Target:.default Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:24.597782 108763 storage/replica_proposal.go:222  [n1,s1,r20/1:/Table/{23-50}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119124.594238335,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:25.436658 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 52, info: {Target:d Config: User:root}
I180801 10:25:26.305108 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 54, info: {Target:d.t Config: User:root}
I180801 10:25:27.010313 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 54, info: {Target:d.t@primary Config: User:root}
I180801 10:25:27.608585 108729 storage/replica_proposal.go:222  [n1,s1,r16/1:/Table/{19-20}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119127.603478303,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:27.793051 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 54, info: {Target:d.t.p0 Config: User:root}
I180801 10:25:28.234878 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 52, info: {Target:d Config: User:root}
I180801 10:25:28.795450 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "remove_zone_config", target: 54, info: {Target:d.t Config: User:root}
I180801 10:25:29.067993 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 0, info: {Target:.default Config:gc: {ttlseconds: 90000} User:root}
I180801 10:25:29.502812 108805 server/status/runtime.go:424  [n1] runtime stats: 983 MiB RSS, 186 goroutines, 20 MiB/59 MiB/99 MiB GO alloc/idle/total, 36 MiB/83 MiB CGO alloc/total, 656.97cgo/sec, 0.38/0.02 %(u/s)time, 0.01 %gc (3x)
I180801 10:25:29.610197 108733 storage/replica_proposal.go:222  [n1,s1,r8/1:/Table/1{1-2}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119129.606778516,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:29.917697 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t.p0 Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:30.420298 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t.p1 Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:30.611952 108752 storage/replica_proposal.go:222  [n1,s1,r19/1:/Table/2{2-3}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119130.608413715,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:31.245750 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t@primary Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:31.613322 108749 storage/replica_proposal.go:222  [n1,s1,r12/1:/Table/1{5-6}] new range lease repl=(n1,s1):1 seq=3 start=1533119109.358915335,0 epo=1 pro=1533119131.609934191,0 following repl=(n1,s1):1 seq=2 start=1533119109.358915335,0 exp=1533119118.470315403,0 pro=1533119109.470407160,0
I180801 10:25:31.737024 109262 sql/event_log.go:126  [n1,client=127.0.0.1:38738,user=root] Event: "set_zone_config", target: 54, info: {Target:d.t.p0 Config:gc: {ttlseconds: 42} User:root}
I180801 10:25:31.789211 108600 util/stop/stopper.go:537  quiescing; tasks left:
1      ts.poller: poll
1      [async] transport racer
I180801 10:25:31.789739 108828 kv/transport_race.go:67  transport race promotion: ran 10 iterations on up to 1992 requests
I180801 10:25:31.789997 108600 util/stop/stopper.go:537  quiescing; tasks left:
1      ts.poller: poll
W180801 10:25:33.523468 108806 ts/db.go:171  [n1,ts-poll] error writing time series data: node unavailable; try another peer
--- FAIL: TestValidIndexPartitionSetShowZones (25.36s)
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX d.t@primary': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX d.t@primary': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX d.t@primary': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		52, d, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX d.t@primary': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR TABLE d.t': expected:
		53, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX d.t@primary': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX d.t@primary': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ALL ZONE CONFIGURATIONS': expected:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		53, d.t.p1, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		0, .default, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 90000
		num_replicas: 1
		constraints: []
		, ��@��� "��(
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		54, d.t.p1, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE d.t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p1 OF TABLE d.t': expected:
		53, d.t.p1, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p1, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR INDEX "primary"': expected:
		53, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t@primary, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
	sql_runner.go:157: query 'EXPERIMENTAL SHOW ZONE CONFIGURATION FOR PARTITION p0 OF TABLE t': expected:
		53, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		
		got:
		54, d.t.p0, range_min_bytes: 1048576
		range_max_bytes: 67108864
		gc:
		  ttlseconds: 42
		num_replicas: 1
		constraints: []
		, ��@��� "*(
		

```

--- a/pkg/ccl/backupccl/backup_test.go
+++ b/pkg/ccl/backupccl/backup_test.go
@@ -444,21 +444,11 @@ func TestBackupRestoreSystemJobs(t *testing.T) {
 	sanitizedFullDir := localFoo + "/full"
 	fullDir := sanitizedFullDir + "?moarSecretsHere"
 
-	backupDatabaseID, err := sqlutils.QueryDatabaseID(sqlDB.DB, "data")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	backupTableID, err := sqlutils.QueryTableID(sqlDB.DB, "data", "bank")
-	if err != nil {
-		t.Fatal(err)
-	}
+	backupDatabaseID := sqlutils.QueryDatabaseID(t, sqlDB.DB, "data")
+	backupTableID := sqlutils.QueryTableID(t, sqlDB.DB, "data", "bank")
 
 	sqlDB.Exec(t, `CREATE DATABASE restoredb`)
-	restoreDatabaseID, err := sqlutils.QueryDatabaseID(sqlDB.DB, "restoredb")
-	if err != nil {
-		t.Fatal(err)
-	}
+	restoreDatabaseID := sqlutils.QueryDatabaseID(t, sqlDB.DB, "restoredb")
 
 	// We create a full backup so that, below, we can test that incremental
 	// backups sanitize credentials in "INCREMENTAL FROM" URLs.
@@ -553,10 +543,7 @@ func checkInProgressBackupRestore(
 
 	sqlDB.Exec(t, `CREATE DATABASE restoredb`)
 
-	backupTableID, err := sqlutils.QueryTableID(sqlDB.DB, "data", "bank")
-	if err != nil {
-		t.Fatal(err)
-	}
+	backupTableID := sqlutils.QueryTableID(t, sqlDB.DB, "data", "bank")
 
 	do := func(query string, check inProgressChecker) {
 		jobDone := make(chan error)
@@ -791,10 +778,7 @@ func TestBackupRestoreResume(t *testing.T) {
 		restoreDir := "nodelocal:///restore"
 		sqlDB.Exec(t, `BACKUP DATABASE DATA TO $1`, restoreDir)
 		sqlDB.Exec(t, `CREATE DATABASE restoredb`)
-		restoreDatabaseID, err := sqlutils.QueryDatabaseID(sqlDB.DB, "restoredb")
-		if err != nil {
-			t.Fatal(err)
-		}
+		restoreDatabaseID := sqlutils.QueryDatabaseID(t, sqlDB.DB, "restoredb")
 		restoreTableID, err := sql.GenerateUniqueDescID(ctx, tc.Servers[0].DB())
 		if err != nil {
 			t.Fatal(err)

--- a/pkg/ccl/changefeedccl/helpers_test.go
+++ b/pkg/ccl/changefeedccl/helpers_test.go
@@ -707,10 +707,7 @@ func forceTableGC(
 	database, table string,
 ) {
 	t.Helper()
-	tblID, err := sqlutils.QueryTableID(sqlDB.DB, database, table)
-	if err != nil {
-		t.Fatal(err)
-	}
+	tblID := sqlutils.QueryTableID(t, sqlDB.DB, database, table)
 
 	tblKey := roachpb.Key(keys.MakeTablePrefix(tblID))
 	gcr := roachpb.GCRequest{

--- a/pkg/ccl/partitionccl/zone_test.go
+++ b/pkg/ccl/partitionccl/zone_test.go
@@ -50,7 +50,8 @@ func TestValidIndexPartitionSetShowZones(t *testing.T) {
 	partialZoneOverride := *config.NewZoneConfig()
 	partialZoneOverride.GC = &config.GCPolicy{TTLSeconds: 42}
 
-	dbDescID := uint32(keys.MinNonPredefinedUserDescID)
+	dbID := sqlutils.QueryDatabaseID(t, sqlDB.DB, "d")
+	tableID := sqlutils.QueryTableID(t, sqlDB.DB, "d", "t")
 
 	defaultRow := sqlutils.ZoneRow{
 		ID:           keys.RootNamespaceID,
@@ -63,54 +64,54 @@ func TestValidIndexPartitionSetShowZones(t *testing.T) {
 		Config:       zoneOverride,
 	}
 	dbRow := sqlutils.ZoneRow{
-		ID:           dbDescID,
+		ID:           dbID,
 		CLISpecifier: "d",
 		Config:       zoneOverride,
 	}
 	tableRow := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t",
 		Config:       zoneOverride,
 	}
 	primaryRow := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t@primary",
 		Config:       zoneOverride,
 	}
 	p0Row := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t.p0",
 		Config:       zoneOverride,
 	}
 	p1Row := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t.p1",
 		Config:       zoneOverride,
 	}
 
 	// Partially filled config rows
 	partialDbRow := sqlutils.ZoneRow{
-		ID:           dbDescID,
+		ID:           dbID,
 		CLISpecifier: "d",
 		Config:       partialZoneOverride,
 	}
 	partialTableRow := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t",
 		Config:       partialZoneOverride,
 	}
 	partialPrimaryRow := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t@primary",
 		Config:       partialZoneOverride,
 	}
 	partialP0Row := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t.p0",
 		Config:       partialZoneOverride,
 	}
 	partialP1Row := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t.p1",
 		Config:       partialZoneOverride,
 	}

--- a/pkg/sql/ambiguous_commit_test.go
+++ b/pkg/sql/ambiguous_commit_test.go
@@ -159,10 +159,7 @@ func TestAmbiguousCommit(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		tableID, err := sqlutils.QueryTableID(sqlDB, "test", "t")
-		if err != nil {
-			t.Fatal(err)
-		}
+		tableID := sqlutils.QueryTableID(t, sqlDB, "test", "t")
 		tableStartKey.Store(keys.MakeTablePrefix(tableID))
 
 		// Wait for new table to split & replication.

--- a/pkg/sql/zone_test.go
+++ b/pkg/sql/zone_test.go
@@ -71,14 +71,17 @@ func TestValidSetShowZones(t *testing.T) {
 		CLISpecifier: "system.jobs",
 		Config:       zoneOverride,
 	}
-	dbDescID := uint32(keys.MinNonPredefinedUserDescID)
+
+	dbID := sqlutils.QueryDatabaseID(t, sqlDB.DB, "d")
+	tableID := sqlutils.QueryTableID(t, sqlDB.DB, "d", "t")
+
 	dbRow := sqlutils.ZoneRow{
-		ID:           dbDescID,
+		ID:           dbID,
 		CLISpecifier: "d",
 		Config:       zoneOverride,
 	}
 	tableRow := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t",
 		Config:       zoneOverride,
 	}
@@ -99,14 +102,13 @@ func TestValidSetShowZones(t *testing.T) {
 		CLISpecifier: "system.jobs",
 		Config:       partialZoneOverride,
 	}
-	dbDescID = uint32(keys.MinNonPredefinedUserDescID)
 	partialDbRow := sqlutils.ZoneRow{
-		ID:           dbDescID,
+		ID:           dbID,
 		CLISpecifier: "d",
 		Config:       partialZoneOverride,
 	}
 	partialTableRow := sqlutils.ZoneRow{
-		ID:           dbDescID + 1,
+		ID:           tableID,
 		CLISpecifier: "d.t",
 		Config:       partialZoneOverride,
 	}

--- a/pkg/testutils/sqlutils/table_id.go
+++ b/pkg/testutils/sqlutils/table_id.go
@@ -16,23 +16,24 @@ package sqlutils
 
 import (
 	gosql "database/sql"
+	"testing"
 )
 
 // QueryDatabaseID returns the database ID of the specified database using the
 // system.namespace table.
-func QueryDatabaseID(sqlDB *gosql.DB, dbName string) (uint32, error) {
+func QueryDatabaseID(t testing.TB, sqlDB *gosql.DB, dbName string) uint32 {
 	dbIDQuery := `SELECT id FROM system.namespace WHERE name = $1 AND "parentID" = 0`
 	var dbID uint32
 	result := sqlDB.QueryRow(dbIDQuery, dbName)
 	if err := result.Scan(&dbID); err != nil {
-		return 0, err
+		t.Fatal(err)
 	}
-	return dbID, nil
+	return dbID
 }
 
 // QueryTableID returns the table ID of the specified database.table
 // using the system.namespace table.
-func QueryTableID(sqlDB *gosql.DB, dbName, tableName string) (uint32, error) {
+func QueryTableID(t testing.TB, sqlDB *gosql.DB, dbName, tableName string) uint32 {
 	tableIDQuery := `
  SELECT tables.id FROM system.namespace tables
    JOIN system.namespace dbs ON dbs.id = tables."parentID"
@@ -41,7 +42,7 @@ func QueryTableID(sqlDB *gosql.DB, dbName, tableName string) (uint32, error) {
 	var tableID uint32
 	result := sqlDB.QueryRow(tableIDQuery, dbName, tableName)
 	if err := result.Scan(&tableID); err != nil {
-		return 0, err
+		t.Fatal(err)
 	}
-	return tableID, nil
+	return tableID
 }

